### PR TITLE
CAT-1295 Miscalculation with decimal places

### PR DESCRIPTION
**Pull request by bernadettespieler**

### CODE REVIEW

**Comment by Bioxar** [comment]
ok to test

**Comment by thmq** [comment]
retest this please

**Comment by thmq** [comment]
retest this please

**Comment by thmq** [comment]
retest this please

File: catroid/src/org/catrobat/catroid/formulaeditor/Formula.java [hunk 1]
```diff
@@ -279,7 +279,63 @@ public String getResultForComputeDialog(Context context) {
...
 			}
-			return String.valueOf(interpretationResult);
+			return DoubleFormatter.roundDoubleToDecimals(interpretationResult, 8);
+		}
+	}
+
+	public static class DoubleFormatter {
+		private static final int MIN_DECIMALS = 0x01;
+		private static final int MAX_DECIMALS = 0xF;
+		private static final int NO_INDEX = 0xFFFFFFFF;
```
Comment (robertpainsi) [inline catroid/src/org/catrobat/catroid/formulaeditor/Formula.java]: Please make human readable :wink: 1, 15, -1

**Comment by robertpainsi** [comment]
Please write tests! :scream_cat:

**Pull request closed without merging**